client/src/components/TestSuite/TestSuiteTree/TestSuiteTree.tsx

```diff
@@ -56,6 +56,8 @@ const TestSuiteTreeComponent: FC<TestSuiteTreeProps> = ({
       />
     ));
 
+    console.log(testSuite);
```

Comment:
Just tagging in case you forgot to remove this. If there is a good reason to keep it I don't mind.

Reply:
Good catch!